database/migrations/2017_01_18_193205_add_log_columns_to_events_table.php

```diff
@@ -0,0 +1,45 @@
+<?php
+
+use Illuminate\Database\Schema\Blueprint;
+use Illuminate\Database\Migrations\Migration;
+
+class AddLogColumnsToEventsTable extends Migration
+{
+    /**
+     * Run the migrations.
+     *
+     * @return void
+     */
+    public function up()
+    {
+        Schema::table('events', function (Blueprint $table) {
+            $table->integer('quantity')->nullable()->after('submission_type');
+            $table->integer('quantity_pending')->nullable()->after('quantity');
+            $table->text('why_participated')->nullable()->after('quantity_pending');
+            $table->text('caption')->nullable()->after('why_participated');
+            $table->text('status')->nullable()->after('caption');
+            $table->text('source')->nullable()->after('status');
+            $table->ipAddress('remote_addr')->nullable()->after('source');
+            $table->text('reason')->nullable()->after('remote_addr');
```

Comment:
I think everything looks good! I just don't know what this column is though, can you remind me?